src/main/java/org/skriptlang/skript/bukkit/itemcomponents/blocking/elements/ExprSecBlankBlockComp.java

```diff
@@ -0,0 +1,119 @@
+package org.skriptlang.skript.bukkit.itemcomponents.blocking.elements;
+
+import ch.njol.skript.config.SectionNode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Example;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SectionExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.Trigger;
+import ch.njol.skript.lang.TriggerItem;
+import ch.njol.skript.lang.util.SectionUtils;
+import ch.njol.skript.registrations.EventValues;
+import ch.njol.skript.variables.Variables;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.itemcomponents.blocking.BlockingExperimentalSyntax;
+import org.skriptlang.skript.bukkit.itemcomponents.blocking.BlockingWrapper;
+import org.skriptlang.skript.registration.SyntaxInfo;
+import org.skriptlang.skript.registration.SyntaxRegistry;
+
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+@Name("Blank Blocking Component")
+@Description("""
+	Gets a blank blocking component.
```

Comment:
need to mention the defaults! Also not sure this should be "blank" if it's full of default values?